WalletWasabi.Gui/Tabs/WalletManager/GenerateWalletSuccessViewModel.cs

```diff
@@ -1,27 +1,21 @@
 ﻿using NBitcoin;
 using ReactiveUI;
-using System;
-using System.Reactive.Disposables;
 using WalletWasabi.Gui.ViewModels;
 
 namespace WalletWasabi.Gui.Tabs.WalletManager
 {
-	internal class GenerateWalletSuccessViewModel : CategoryViewModel, IDisposable
-	{
-		private CompositeDisposable Disposables { get; }
-
+	internal class GenerateWalletSuccessViewModel : CategoryViewModel
+	{ 
 		private string _mnemonicWords;
 
 		public GenerateWalletSuccessViewModel(WalletManagerViewModel owner, Mnemonic mnemonic) : base("Wallet Generated Successfully!")
 		{
-			Disposables = new CompositeDisposable();
-
 			_mnemonicWords = mnemonic.ToString();
 
 			ConfirmCommand = ReactiveCommand.Create(() =>
 			{
 				owner.SelectTestPassword();
-			}).DisposeWith(Disposables);
+			});
```

Comment:
Why not dispose ReactiveCommands?

Reply:
The reason is:

- Reactive command doesnt have any unmanaged resources.
- Dispose in Reactive command **doesnt** mean release unmanaged resources, it simply means unsubscribe.
- Reactive command is on the same object that is subscribing, so GC will handle everything.

So no memory leak here.


See this comment from the author of RxUI...
https://github.com/reactiveui/ReactiveUI/issues/20#issuecomment-1324201